new-app/src/App/runGame/OTPInputGuess.tsx

```diff
@@ -52,9 +59,35 @@ const OTPInputGuess = ({guess, setGuess}: {guess: any[], setGuess: React.Dispatc
     submitGuess();
   };
 
+  const buildDigitStyle = (hint?: string): string => {
+    switch (hint) {
+      case ("miss"):
+        return "box-output border-[#c1292e] text-[#c1292e]"
+      case ("key"):
+        return "box-output border-hacky-green"
+    }
+    return "box-output border-gray-600"
+  }
+
 
   return (
     <div>
+      <div>
+        <div className="flex items-center justify-center">
+          <div className={ClassNames(buildDigitStyle(hintArray[0]))}>
```

Comment:
This looks like a good place for a map! We could map through the hintArray to build up all these elements.